frappe/public/js/frappe/form/layout.js

```diff
@@ -218,23 +250,33 @@ frappe.ui.form.Layout = class Layout {
 		this.column = null;
 	}
 
-	make_column (df) {
-		this.column = new frappe.ui.form.Column(this.section, df);
+	make_column(df) {
+		this.column = new Column(this.section, df);
 		if (df && df.fieldname) {
 			this.fields_list.push(this.column);
 		}
 	}
 
-	refresh (doc) {
-		var me = this;
+	make_tab(df) {
+		this.tab = new Tab(this, df);
+
+		if (df) {
+			this.fields_dict[df.fieldname] = this.tab;
+			this.fields_list.push(this.tab);
+		}
+
```

Comment:
add this line to handle no immediate section break under tab break

this.section = null;